src/components/dialogs/network-modification/composite-modification/composite-modification-edition-dialog.tsx

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { FunctionComponent, SyntheticEvent, useEffect, useState } from 'react';
+import { useParameterState } from '../../use-parameters-dialog';
+import { PARAM_LANGUAGE } from '../../../../utils/config-params';
+import {
+    CustomMuiDialog,
+    FieldConstants,
+    NetworkModificationMetadata,
+    NO_SELECTION_FOR_COPY,
+    unscrollableDialogStyles,
+    useModificationLabelComputer,
+    useSnackMessage,
+    yupConfig as yup,
+} from '@gridsuite/commons-ui';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { getCompositeModificationContent, saveCompositeModification } from '../../../../utils/rest-api';
+import CompositeModificationEditionForm from './composite-modification-edition-form';
+import { List, ListItem } from '@mui/material';
+import { useIntl } from 'react-intl';
+import Divider from '@mui/material/Divider';
+import Box from '@mui/material/Box';
+import { setSelectionForCopy } from '../../../../redux/actions';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppState } from '../../../../redux/reducer';
+
+const schema = yup.object().shape({
+    [FieldConstants.NAME]: yup.string().trim().required('nameEmpty'),
+});
+
+const emptyFormData = (name?: string) => ({
+    [FieldConstants.NAME]: name,
+});
+
+interface FormData {
+    [FieldConstants.NAME]: string;
+}
+
+interface CompositeModificationEditionDialogProps {
+    compositeModificationId: string;
+    open: boolean;
+    onClose: (event?: SyntheticEvent) => void;
+    titleId: string;
+    name: string;
+    broadcastChannel: BroadcastChannel;
+}
+
+export const CompositeModificationEditionDialog: FunctionComponent<CompositeModificationEditionDialogProps> = ({
+    compositeModificationId,
+    open,
+    onClose,
+    titleId,
+    name,
+    broadcastChannel,
+}: Readonly<CompositeModificationEditionDialogProps>) => {
+    const intl = useIntl();
+    const [languageLocal] = useParameterState(PARAM_LANGUAGE);
+    const [isFetching, setIsFetching] = useState(!!compositeModificationId);
+    const { snackError } = useSnackMessage();
+    const selectionForCopy = useSelector((state: AppState) => state.selectionForCopy);
+    const [modifications, setModifications] = useState<NetworkModificationMetadata[]>([]);
```

Comment:
useState<NetworkCompositeModificationMetadata[]>([]);

Reply:
Actually no : those are the "metadata" from regular network modifications, not from composite network modification.